feat: more Realtime docs (#12424)
supabase/supabase: apps/docs/pages/guides/realtime/channels.mdx

import Layout from '~/layouts/DefaultGuideLayout'

export const meta = {
  id: 'channels',
  title: 'Realtime Channels',
  description: 'WebSockets are Channels on Supabase Realtime',
  sidebar_label: 'Channels',
}

> Channels are a `topic` you define when you initialize your Supabase Realtime client. Everyone connected to the same Channel will receive the same messages.

Reviewer: ```suggestion
You can uniquely reference a channel by its `topic` you define when you initialize your Supabase Realtime client. Everyone connected to the same Channel topic will receive the same messages.
```
